src/utils.js

```diff
@@ -125,19 +125,19 @@ export const MAX_BODIES_SIZE = mbs >= 1 && mbs <= 15 ? mbs * 1024 * 1024 : 15 *
 const appendText = config.api.truncateAppend
 const appendTextLength = Buffer.byteLength(appendText)
 
-export function enforceMaxBodiesSize (ctx, tx) {
+export function enforceMaxBodiesSize (ctx, body) {
   let enforced = false
 
   // running total for all bodies
   if ((ctx.totalBodyLength == null)) { ctx.totalBodyLength = 0 }
 
-  let len = Buffer.byteLength(tx.body)
+  let len = Buffer.byteLength(body)
```

Comment:
Lots of the tests are failing here. This code doesn't cater for when the body is type string. Should there be a check of string/Array-/Buffer-type to decide how to get the length/byteLength correctly?